MapboxCoreNavigation.podspec

```diff
@@ -42,7 +42,7 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxNavigationNative", "~> 3.1"
   s.dependency "MapboxDirections.swift", "~> 0.24.0"
-  s.dependency "MapboxMobileEvents", "~> 0.5"
+  s.dependency "MapboxMobileEvents", "~> 0.5.0"
```

Comment:
@frederoni this should actually be `0.5` not `0.5.0`. Current `0.5` doesn't contain the updated strings reflected in `0.6`

Reply:
`0.5` actually breaks the `xcode-10-cocoapods-update` bot. Check the status on the first commit. Are you saying we should upgrade to `0.6`?

Reply:
@frederoni yes, we should upgrade at this point.